accname/index.html

```diff
@@ -143,6 +143,11 @@
         xref: ["dom", "core-aam-1.2", "wai-aria-1.2", "infra"],
       };
     </script>
+    <style>
+      .ol--upperAlpha{ list-style:upper-alpha; } /* replaces: ol ol */
+      .ol--lowerRoman { list-style:lower-roman; } /* replaces: ol ol ol */
+      .ol--lowerAlpha { list-style:lower-alpha; } /* replaces: ol ol ol ol */
```

Comment:
Can't we do something like putting the class "computation_algorithm" [on the rool `ol`](https://github.com/w3c/aria/pull/2410/commits/f9522dc93db95b251cc6f100b2cfd17ee1274b4a#diff-c6c1825243b3e3b856bcd53c55febe98bf24dd51e27debb8ec71d71fa98a9a23L433) and then do:

```
.computation_algorithm ol ol { list-style:upper-alpha;  }
.computation_algorithm ol ol ol { list-style:lower-roman; }
.computation_algorithm ol ol ol ol {l ist-style:lower-alpha; }
```

And remove those custom classes on the nest `ol`s, so no one has to worry about adding then when they make a nested list in a computation?

Reply:
Two minor notes: 

- if `.computation_algorithm` goes on an OL, this needs to drop one ol from each line.
- then these styles would still affect the two lists in the Note and Example

As I said on the call, I have no strong feelings either way. But we can also work this out later. (I'm a little worried about merge conflicts as I'm touching all the specs.)

Reply:
oh right. I forgot about the Notes and Example lists. Ok fine to land and change our minds later, if need be.